src/builtin.js

```diff
@@ -536,10 +536,9 @@ Sk.builtin.int2str_ = function helper_ (x, radix, prefix) {
     var str = "";
     if (x instanceof Sk.builtin.lng) {
         suffix = "";
-        if (radix !== 2) {
+        if (radix !== 2 && (!(Sk.__future__.python3))) {
```

Comment:
there's an `L` suffix option on `__future__`

Reply:
I asked about this in #905, but I guess I wasn't specific enough.  What's the point of carrying around the specific flags anymore (except those that are in CPython 2 to allow you to turn on some Python 3 features individually)?

I can change this, but I'd prefer to delete the L suffix future flag and make it use the Python3 future flag in the other places it's used.

Reply:
Yeah that's fine with me too.